.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+ignore = E501,F999
+max-line-length = 100
+exclude = .git,__pycache__,venv
```

Comment:
Ooh, I had no idea you could do a `.flake8` file. That's great. Can you also tell it to specifically include some files that don't end with `.py` (such as `scan` and `gather`)?